packages/adapter-cloudflare-workers/index.js

```diff
@@ -149,29 +149,33 @@ export default function ({ config = 'wrangler.toml', platformProxy = {} } = {})
 			builder.writePrerendered(bucket_dir);
 		},
 
-		async emulate() {
-			const proxy = await getPlatformProxy(platformProxy);
-			const platform = /** @type {App.Platform} */ ({
-				env: proxy.env,
-				context: proxy.ctx,
-				caches: proxy.caches,
-				cf: proxy.cf
-			});
+		emulate() {
+			const getting_platform = (async () => {
```

Comment:
this could use a comment explaining why it's wrapped this way

Reply:
I'm not quite sure how to phrase the reasoning but I'll try to add a comment.